api-operator/pkg/envoy/deleteAPI.go

```diff
@@ -182,7 +182,9 @@ func deleteAPISwagger(config *corev1.ConfigMap, token string, endpoint string) e
 	if resp.StatusCode() == http.StatusOK {
 		return nil
 	} else if resp.StatusCode() == http.StatusNotFound {
-		logDelete.Info("API does not exist")
+		logDelete.Error(nil, "API does not exist - " + apiName + " - " + apiVersion)
```

Comment:
shall use key-value pairs for logs
eg:
```suggestion
		logDelete.Error(nil, "API not found", "api_name", apiName, "api_version",  apiVersion)
```

Reply:
+1. Fixed.